definitions/checks/foreman_proxy/check_tftp_storage.rb

```diff
@@ -22,10 +22,11 @@ def run
     end
 
     def old_files_from_tftp_boot
-      Dir.entries(tftp_boot_directory).map do |file|
+      initrd = File.join(tftp_boot_directory, '/*-initrd.img')
+      vmlinuz = File.join(tftp_boot_directory, '/*-vmlinuz')
+      Dir.glob([initrd, vmlinuz]).map do |file|
```

Comment:
```suggestion
      Dir.glob("#{tftp_boot_directory}*-{vmlinuz,initrd.img}").map do |file|
```
@patilsuraj767, what you think?